corehq/motech/openmrs/serializers.py

```diff
@@ -54,6 +57,15 @@ def to_omrs_datetime(value):
         return value.strftime('%Y-%m-%dT%H:%M:%S.{f}{z}'.format(f=micros, z=tz))
 
 
+def to_omrs_boolean(value):
+    if (
+        isinstance(value, six.string_types)
+        and value.lower() in ('false', '0')
+    ):
+        return False
+    return bool(value)
```

Comment:
FYI, there's also https://github.com/dimagi/commcare-hq/blob/9ad6aa8f1112ba571d38529bfa47b4659e450182/corehq/ex-submodules/dimagi/utils/parsing.py#L8-L24

Reply:
Though if you have a specific set of anticipated values and don't want the additional flexibilty, this may make more sense.